### DISCUSSION TITLE
可以试着增加一个学生自己上传资料的窗口吗

### DISCUSSION THREAD

**Discussion by 1BIMU** [byrdocs/byrdocs-archive#1] [Ideas]
这样可以保持资料的随时更新~

**Comment by chisato2233** [comment]
应该可以给 https://github.com/byrdocs/byrdocs-archive 提交pr（？

**Reply by cppHusky** [reply]
> 应该可以给 https://github.com/byrdocs/byrdocs-archive 提交pr（？

[archive](https://github.com/byrdocs/byrdocs-archive) 最多只有元信息表格，这个 Repository 本身并不存储 BYR Docs 的文件。
所以提交PR的思路应该是行不通的。

**Comment by cppHusky** [comment]
> 这样可以保持资料的随时更新~

您好，
现阶段，本站的资料整理工作均由本人完成。新加入的资料需要经过本人审核文件质量，并记录文件信息。本人亦承担合并/拆分文件、添加书籍封面、比较重复文件质量并取舍等工作。
目前的工作流程只支持少数几个专门人士完成资料整理，不能支持面向大量用户。因此，**本站目前不对用户设计上传资料的功能，在可预见的将来也不会支持此功能**，请您谅解。
如果有文件想要提供给本站，欢迎通过电子邮件等方式发送您的文件。您也可以加入QQ群 `829649976` 上传您的文件。本人会在收到文件后予以处理。

**Comment by YouXam** [comment]
现已支持贡献文件，欢迎参与，请参阅[贡献指南](https://github.com/byrdocs/byrdocs-archive/blob/master/CONTRIBUTING.md)。
